src/actions/README.md

```diff
@@ -8,7 +8,7 @@ All actions in perf.html are fully typed using [Flow](https://flow.org/). These
 
 # Preferred practices for actions
 
-Actions in perf.html are kept relatively simple. There are two types of action creators, either functions that return an `Action` or a function that returns a `ThunkAction` (see [Redux Thunk](https://github.com/gaearon/redux-thunk) for additional reading.) The action creators that return `Action`s are quite simple and work as one expects. `ThunkAction`s on the other hand have a few restrictions in this project. `ThunkAction`s should not directly access the global state. Instead, the components calling the action creators should use selectors to fetch the required information, and then pass it into action creator function.
+Actions in perf.html are kept relatively simple. There are two types of action creators in this project–action creators that return an `Action` or an action creator that returns a `ThunkAction` (see [Redux Thunk](https://github.com/gaearon/redux-thunk) for additional reading.) If an action creator needs access to different parts of the state, then it may be tempting to use the `getState` parameter from a `ThunkAction`. However, at this time we prefer not to directly access the state in a `ThunkAction`. Instead, the connected component should look up the value using a selector, then pass it into the action creator as a parameter. 
```

Comment:
> If an action creator needs access to different parts of the state, then it may be tempting to use the `getState` parameter from a `ThunkAction`. However, at this time we prefer not to directly access the state in a `ThunkAction`.

=> It's not only for `ThunkAction`s (which are async actions) but for all type of actions. I think ?